epictrack-web/src/components/insights/Work/Tabs/Partners/charts/WorkList.tsx

```diff
@@ -38,9 +38,19 @@ const WorkList = () => {
       showNotification("Error fetching works", { type: "error" });
     }
   }, [error]);
-
   const federalInvolvements = useMemo(() => {
-    return Array.from(new Set(works.map((w) => w?.federal_involvement?.name)));
+    return Array.from(
+      new Set(
+        [...works]
```

Comment:
why the spread operator? why not just works instead of [...works].

Reply:
The sort directly tries to edit the variable works and causes error "Cannot assign to read only property '0' of object '[object Array]'". Spread is required to create a copy and perform sort

Reply:
oh cool, fair enough

Reply:
Ok I see, fair